### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by AndrzejBulatek**

### CODE REVIEW

**Review by jacek-bulatek — DISMISSED** [review]
in LevelManager class constructor change LvlNo to LevelNo

File: app/src/main/java/com/example/nockanakalinowej/LevelMenager.java [hunk 1]
```diff
@@ -16,19 +16,19 @@
...
     int space;
-    int kflamountx;
-    int kflamounty;
+    int tileAmountX;
+    int tileAmountY;
     int width;
     int height;
     int[] idList;
 
-    public LvlMng(Level context, int LvlNo){
```
Comment (jacek-bulatek) [inline app/src/main/java/com/example/nockanakalinowej/LevelMenager.java]: change LvlNo to LevelNo

File: app/src/main/java/com/example/nockanakalinowej/LevelMenager.java [hunk 2]
```diff
@@ -16,19 +16,19 @@
...
+    public LevelMenager(Level context, int LvlNo){
```
Comment (jacek-bulatek) [inline app/src/main/java/com/example/nockanakalinowej/LevelMenager.java]: change LvlNo to LevelNo

File: app/src/main/java/com/example/nockanakalinowej/LevelMenager.java [hunk 3]
```diff
@@ -50,11 +50,11 @@ public LvlMng(Level context, int LvlNo){
         setLayout(setKafels(LvlNo));
```
Comment (jacek-bulatek) [inline app/src/main/java/com/example/nockanakalinowej/LevelMenager.java]: same here

**Review by jacek-bulatek — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/nockanakalinowej/LevelMenager.java [hunk 4]
```diff
@@ -50,11 +50,11 @@ public LvlMng(Level context, int LvlNo){
...
     }
 
-    protected void setLayout(int kafelNo){
-        final ImageButton[] buttArr = new ImageButton[kafelNo];
-        int edge = kafelEdge(kflamountx, kflamounty);
-        lvlLayout = new ConstraintLayout(window);
-        lvlLayout.setId(R.id.Level_layout);
+    protected void setLayout(int tileAmount){
+        final ImageButton[] buttArr = new ImageButton[tileAmount];
```
Comment (mbulatek) [inline app/src/main/java/com/example/nockanakalinowej/LevelMenager.java]: Suggest to change to buttonsArray

File: app/src/main/java/com/example/nockanakalinowej/LevelMenager.java (hunk 2 above)
Comment (AndrzejBulatek) [reply app/src/main/java/com/example/nockanakalinowej/LevelMenager.java]: done

File: app/src/main/java/com/example/nockanakalinowej/LevelMenager.java (hunk 3 above)
Comment (AndrzejBulatek) [reply app/src/main/java/com/example/nockanakalinowej/LevelMenager.java]: done

File: app/src/main/java/com/example/nockanakalinowej/LevelMenager.java (hunk 4 above)
Comment (AndrzejBulatek) [reply app/src/main/java/com/example/nockanakalinowej/LevelMenager.java]: and done

**Comment by AndrzejBulatek** [comment]
tiles don't rescale simultanously anymore after refactoring

**Review by AndrzejBulatek — COMMENTED** [review]
Bravo Andrzej, very well done. Honestly you shown who's the real mvp in this project.

File: app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java [hunk 5]
```diff
@@ -0,0 +1,86 @@
+package com.example.nockanakalinowej;
+
+import android.animation.Animator;
+import android.animation.AnimatorListenerAdapter;
+import android.animation.AnimatorSet;
+import android.animation.ObjectAnimator;
+import android.view.View;
+
```
Comment (mbulatek) [inline app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java]: Why
import android.view.animation.AnimationSet;
import android.widget.ImageButton;
was removed?

File: app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+        ObjectAnimator translationYTile1 = ObjectAnimator.ofFloat(tile2, "translationY", tile1.getY()-tile2.getY());
+        translationYTile1.setDuration(1000);
+        ObjectAnimator scaleXTile2 = ObjectAnimator.ofFloat(tile2, "scaleX", 1.2f);
+        scaleXTile2.setDuration(1000);
+        ObjectAnimator scaleYTile2 = ObjectAnimator.ofFloat(tile2, "scaleY", 1.2f);
+        scaleYTile2.setDuration(1000);
+        ObjectAnimator translationXTile2 = ObjectAnimator.ofFloat(tile2, "translationX", tile1.getX()-tile2.getX());
+        translationXTile2.setDuration(1000);
+        ObjectAnimator translationYTilel2 = ObjectAnimator.ofFloat(tile1, "translationY", tile2.getY()-tile1.getY());
+        translationYTilel2.setDuration(1000);
```
Comment (mbulatek) [inline app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java]: translationY**Tilel**2?

File: app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+        ObjectAnimator reScaleXTile1 = ObjectAnimator.ofFloat(tile1, "scaleX", 1f);
+        scaleXTile1.setDuration(1000);
+        ObjectAnimator reScaleYTile1 = ObjectAnimator.ofFloat(tile1, "scaleY", 1f);
+        scaleYTile1.setDuration(1000);
+        ObjectAnimator reScaleXTile2 = ObjectAnimator.ofFloat(tile2, "scaleX", 1f);
+        reScaleXTile2.setDuration(1000);
```
Comment (mbulatek) [inline app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java]: Why did you change scaleXTile2 to reScaleXTile2? These are different objects!

File: app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java [hunk 8]
```diff
@@ -0,0 +1,86 @@
...
+        ObjectAnimator reScaleYTile2 = ObjectAnimator.ofFloat(tile2, "scaleY", 1f);
+        reScaleYTile2.setDuration(1000);
```
Comment (mbulatek) [inline app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java]: Why did you change scaleYTile2 to reScaleYTile2? These are different objects!

**Review by mbulatek — DISMISSED** [review]

File: app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java (hunk 5 above)
Comment (jacek-bulatek) [reply app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java]: 'Cause they're unused

File: app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java (hunk 7 above)
Comment (jacek-bulatek) [reply app/src/main/java/com/example/nockanakalinowej/AnimationMenager.java]: scaleXTile is set above, this is correct place for rescaling, however it should be changed also in lines: 39 and 41

**Review by jacek-bulatek — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/nockanakalinowej/Level.java [hunk 9]
```diff
@@ -23,22 +21,22 @@
...
-        clickedKafelIDs = new int[2];
-        lvlNo = 5;
-        aniMng = new AniMng(this);
-        lvlMng = new LvlMng(this, lvlNo);
-        setContentView(lvlMng.getLayout());
+        clickedTileIDs = new int[2];
+        levelNo = 5;
+        animationMenager = new AnimationMenager(this);
+        levelMenager = new LevelMenager(this, levelNo);
+        setContentView(levelMenager.getLayout());
```
Comment (jacek-bulatek) [inline app/src/main/java/com/example/nockanakalinowej/Level.java]: It's M**a**nager not M**e**nager

File: app/src/main/java/com/example/nockanakalinowej/LevelMenager.java [hunk 10]
```diff
@@ -140,129 +140,129 @@ public void onClick(View view) {
...
             });
         }
     }
 
-    public ConstraintLayout getLayout(){ return lvlLayout;}
+    public ConstraintLayout getLayout(){ return levelLayout;}
 
-    protected int setKafels(int lvlNo){
-        switch (lvlNo) {
+    protected int settiles(int levelNo){
```
Comment (jacek-bulatek) [inline app/src/main/java/com/example/nockanakalinowej/LevelMenager.java]: Should be setTiles in lowerCamelCase

File: app/src/main/res/values/id.xml [hunk 11]
```diff
@@ -7,35 +7,35 @@
...
-    <item name="kafel22" type="id" />
-    <item name="kafel23" type="id" />
-    <item name="kafel24" type="id" />
-    <item name="kafel25" type="id" />
-    <item name="kafel26" type="id" />
-    <item name="kafel27" type="id" />
-    <item name="kafel28" type="id" />
-    <item name="kafel29" type="id" />
-    <item name="kafel30" type="id" />
+    <item name="tile_Field" type="id" />
```
Comment (jacek-bulatek) [inline app/src/main/res/values/id.xml]: dont know what java convention says about this kind of names, but to stay similar to Level_layout this should be Tile_field

**Review by jacek-bulatek — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/nockanakalinowej/LevelMenager.java (hunk 4 above)
Comment (jacek-bulatek) [reply app/src/main/java/com/example/nockanakalinowej/LevelMenager.java]: well buttonArr is kinda different than buttonsArray, but whatever

**Review by jacek-bulatek — APPROVED** [review]
